test/unittests/validation_stack_test.cpp

```diff
@@ -160,7 +180,7 @@ TEST(validation_stack, loop_with_result_stack_underflow)
     EXPECT_THROW_MESSAGE(parse(wasm), validation_error, "stack underflow");
 }
 
-TEST(validation_stack, DISABLED_loop_too_many_results)
+TEST(validation_stack, loop_too_many_results)
```

Comment:
Another test suggestion, not sure if already exists: unreachable insrtuction after `br` that would push extra result, but should be valid, as it's unreachable.

Reply:
This is actually invalid. You cannot have additional items in polymorphic stack.